internal/benchrunner/runners/stream/runner.go

```diff
@@ -430,7 +430,9 @@ func (r *runner) collectBulkRequestBody(indexName, scenarioName string, buf *byt
 	var event map[string]any
 	err = json.Unmarshal(buf.Bytes(), &event)
 	if err != nil {
-		return bulkBodyBuilder, err
+		return bulkBodyBuilder, fmt.Errorf("problem unmarshaling json event: %s\n"+
+			"Outcome of template for scenario '%s' is invalid JSON. Check your benchmark template.",
+			buf.String(), scenarioName)
```

Comment:
Linter doesn't like error strings ended with punctuation.
```
internal/benchrunner/runners/stream/runner.go:433:27: error strings should not end with punctuation or newlines (ST1005)
```
This could be changed to something like this, logging the failed document at the debug level, and including the original error too:
```suggestion
		logger.Debugf("problem found when unmarshalling document: %s", buf.String())
		return bulkBodyBuilder, fmt.Errorf("failed to unmarshal json event, check your benchmark template for scenario %s: %w", scenarioName, err)
```